EssentialApp/EssentialAppTests/Helpers/Image Comments/ImageCommentsViewController+TestHelpers.swift

```diff
@@ -0,0 +1,50 @@
+//
+//  ImageCommentsViewController+TestHelpers.swift
+//  EssentialAppTests
+//
+//  Created by Lukas Bahrle Santana on 01/02/2021.
+//  Copyright © 2021 Essential Developer. All rights reserved.
+//
+
+import UIKit
+import EssentialFeediOS
+
+extension ImageCommentCell{
+	var messageText: String?{ messageLabel.text }
+	var createdAtText: String?{ createdAtLabel.text }
+	var usernameText: String?{ usernameLabel.text }
+}
+
+
+extension ImageCommentsViewController {
```

Comment:
Remove extra line break

```suggestion
}

extension ImageCommentsViewController {
```